src/exceptions/HO100206.ts

```diff
@@ -0,0 +1,20 @@
+import type Exception from "src/types/Exception"
+import { ExceptionCode } from "src/types/ExceptionCode"
+import type { ExceptionGenerator } from "src/types/ExceptionGenerator"
+import { validateASN, validateDummyASN } from "src/use-cases/validations"
+
+const ARREST_SOMMONS_NUMBER_PATH: (string | number)[] =
```

Comment:
Typo: `SOMMONS`

Reply:
Fixed in [9bc5892](https://github.com/ministryofjustice/bichard7-next-core/pull/89/commits/9bc5892be0725d93289ec385b9c059c7b3700205)